vmdb/app/presenters/tree_node_builder.rb

```diff
@@ -52,6 +52,8 @@ def build
       "#{ui_lookup(:table=>object.kind_of?(EmsInfra) ? "ems_infra" : "ems_cloud")}: #{object.name}")
     when ChargebackRate       then generic_node(object.description, "chargeback_rates.png")
     when Condition            then generic_node(object.description, "miq_condition.png")
+    when ConfigurationProfileForeman      then generic_node(object.name, "configuration_profile.png")
+    when ConfiguredSystemForeman          then generic_node(object.hostname, "configured_system.png")
```

Comment:
Same here with the subclass specifics...Nothing else does that so why is that being done here?